crates/pera-core/src/dwallet_mpc/mpc_outputs_verifier.rs

```diff
@@ -61,9 +64,23 @@ impl DWalletMPCOutputsVerifier {
                 .collect(),
             completed_locking_next_committee: false,
             voted_to_lock_committee: HashSet::new(),
+            // Todo (#394): Remove hardcoded network key version
+            network_key_version: if let Some(versions) = epoch_store
+                .get_encryption_of_decryption_key_shares()
+                .unwrap_or(HashMap::new())
+                .get(&(DWalletMPCNetworkKey::Secp256k1 as u8))
+            {
+                versions.len() as u8
+            } else {
+                1
+            },
         }
     }
 
+    pub fn network_key_version(&self) -> u8 {
+        self.network_key_version
```

Comment:
Why are you not simply making this field public?

Reply:
This is treated differently in a later PR